analyses/mwi/dryspells_trigger/exploration/2021_2022_postseason_overview.Rmd

```diff
@@ -0,0 +1,570 @@
+
+During the 2021-2022 season, streaks of dry days were common. Every area in the region experienced a dry spell of at least 7 consecutive dry days and most areas experienced a streak of 10+ days. By contrast, fewer areas (`r round(100*length(cells(static_r[["n_14dplus_streaks"]])) / nrow(season_dates), 1)` % of the region) experienced streaks of 14+ dry days. (Recall that by the AA trigger definition no dry spells occurred in the region.) Had the AA trigger defined a spell as a 14-day streaks of dry days instead, more shocks would have been recorded and even more events had the streak been reduced in duration. If a change in definition were considered, the duration of the streak should be determined to reflect a widespread, severe event that occurs once every 3 to 5 years, the type of shocks of which AA aims to get ahead. 
+
+### Q3: How much rainfall was typical over 14-day periods during the rainy season? 
+
+The definition used in the AA trigger set a maximum of 2mm of cumulative rainfall over at least 14 days, an amount that could be adjusted. The 2021-2022 season analysis revealed that very few (`r round(100*ecdf(in_season$roll_sum_next_14d)(2), 2)`%) periods received 2mm or less. Even when allowing for 5 times more rainfall than the selected definition for dry spell (ie for 10mm over 14 days), the occurrence of periods with so little rainfall was still significantly less than 5% of the periods during the rainy season. It may also point to a reduced ability to detect highly localised dry spells when using averages over larger regions such as adm2 (districts), where an area that received rain within the district may cause the district average to be higher than 2mm despite other areas within the district having received no rain. An analytical approach that considers geographical coverage within a district, rather than an average over a district, may be a more appropriate methodological choice.
+
+Rainfall patterns from the 2021-2022 season as examined under Q2 and Q3 suggest that the AA trigger definition of dry spell only reflect rare and localised events, and may be overly restrictive for AA by not capturing periods of less severe dryness that could still cause significant crop damage and humanitarian needs. Further work would be needed to evaluate alternative definitions to find one that may be better suited for AA. Furthermore, using a less restrictive definition might result in improved predictability and thus, a better performing predictive trigger. This is speculative at the moment and could only be verified through extensive analysis.
```

Comment:
I don't know about this conclusion. Are we sure that it's not that actually dry spells just weren't particularly bad this year, and maybe the focus on dry spell is the problem rather than their definition?